service-api/module/Application/src/DWP/DwpApi/DTO/CitizenRequestDTO.php

```
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\DWP\DwpApi\DTO;
+
+use Application\DWP\DwpApi\DwpApiException;
+use Application\Model\Entity\CaseData;
+
+class CitizenRequestDTO
+{
+    private string $firstName;
+    private string $lastName;
+    private string $dob;
+    private string $postcode;
+    private string $addressLine1;
+    private string $nino;
+
+    public function __construct(
+        protected CaseData $caseData
+    ) {
+        try {
+            /**
+             * @psalm-suppress PossiblyNullPropertyFetch
+             */
+            if ($this->caseData->idMethodIncludingNation->id_method !== 'NATIONAL_INSURANCE_NUMBER') {
+                throw new DwpApiException('Identity method is not a national insurance number');
+            }
+
+            /**
+             * @psalm-suppress PossiblyNullArrayAccess
+             * @psalm-suppress PossiblyNullPropertyFetch
+             * @psalm-suppress PossiblyNullPropertyAssignmentValue
+             */
```

it might be neater to put these in the function docblock rather than repeating them for each variable.
Though I wonder if it would be better to throw an exception if any of them are null as they shouldn't be at this point in the journey.